src/lib/util/Leaderboard.ts

```diff
@@ -1,10 +1,12 @@
-import { Client, Collection } from 'discord.js';
-import { PreciseTimeout } from './PreciseTimeout';
+import { Cache } from '@klasa/cache';
+import { DbSet } from '@lib/structures/DbSet';
+import { Client } from 'discord.js';
 import { Time } from './constants';
+import { PreciseTimeout } from './PreciseTimeout';
 
 /**
  * The Leaderboard singleton class in charge of storing local and global leaderboards
- * @version 2.0.0
+ * @version 2.1.0
```

Comment:
Not that I mind but... huh?

Reply:
Kind of versioning Skyra's sub-modules, lmao